src/main/io/osd.c

```diff
@@ -1275,7 +1275,7 @@ static void osdGetBlackboxStatusString(char * buff)
 
 #ifdef USE_FLASHFS
     case BLACKBOX_DEVICE_FLASH:
-        storageDeviceIsWorking = flashfsIsReady();
+        storageDeviceIsWorking = flashfsIsSupported();
```

Comment:
See comment above.

Reply:
It should be `flashfsIsSupported`, as the purpose of checking here is to determine if the chip exists so that its size can be printed (i.e., not to examine if the chip is ready or not).